tests/test_save.py

```diff
@@ -238,16 +224,16 @@ def test_save_emmo_domain_ontology(
     assert set(
         os.listdir(outputdir / "emmo.info" / "emmo" / "domain" / "chameo")
     ) == {"chameo.rdfxml", "catalog-v001.xml"}
-    assert set(
+    assert not {"isq.rdfxml", "catalog-v001.xml"}.difference(
         os.listdir(outputdir / "emmo.info" / "emmo" / "disciplines")
-    ) == {"isq.rdfxml", "catalog-v001.xml"}
```

Comment:
Why?

Reply:
Because onto.save() stores other files in addition to isq.rdfxml and the catalogue file. Explicitly checking for all created files will fail when new modules are added to EMMO disciplines.